File: world_simulation/air.py
class Air:
    def __init__(self, volume: int = 100) -> None:
        self._volume = volume
    
    def increase(self, volume: int = 1) -> None:
        self._volume += volume
    
    def decrease(self, volume: int = 1) -> None:
        if self._volume >= volume:
            self._volume -= volume
        else:
            raise ValueError("Air volume can't be negative.")
    
    def get_volumne(self) -> int:
        return self._volume
    
    def print_status(self) -> None:
        print(f"Air volume is: {self.get_volumne()}")

File: world_simulation_c/air.c
#include <stdio.h>
#include <stdlib.h>

typedef struct Air {
    int volume;
} Air;

Air * air_init(){
    return (Air*)malloc(sizeof(Air));
}

void air_construct(Air * a, int volume){
    a->volume = volume;
}

void air_destruct(Air * a){
    free(a);
}

void increase(Air * a, int volume){
    a->volume += volume;
}

void decrease(Air * a, int volume){
    a->volume -= volume;
}

void air_print_status(Air * a){
    printf("Air volume is %d.\n", a->volume);
}